process-pull-request.py

```diff
@@ -66,16 +155,18 @@
         else:
             print(get_pr_latest_commit(args[0], opts.repository))
     else:
-        from github import Github
-
         repo_dir = join(SCRIPT_DIR, "repos", opts.repository.replace("-", "_"))
         if exists(repo_dir):
             sys.path.insert(0, repo_dir)
         import repo_config
 
         if not getattr(repo_config, "RUN_DEFAULT_CMS_BOT", True):
             sys.exit(0)
-        gh = Github(login_or_token=get_gh_token(opts.repository), per_page=100)
+        if opts.debug:
+            gh = DebugGithub(login_or_token=get_gh_token(opts.repository), per_page=100)
+        else:
+            gh = Github(login_or_token=get_gh_token(opts.repository), per_page=100)
+
```

Comment:
this change looks bit complex . Is there no other way to achieve this?

Reply:
Didn't think of it, but yes, it should work.